### PR TITLE
fixed #22193: Crash when opening Mixer

### LINKED ISSUE #22193: Crash when opening Mixer (MuseScore 4.4.0.240950301 nightly master, Mac only)
### Issue type

Crash or freeze

### Bug description

_No response_

### Steps to reproduce

1. Open **MuseScore 4.4.0.240950301** on Mac
2. Create score
3. Open Mixer

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/90187801/a359233e-f86d-4872-990a-b674f8d91b51

### MuseScore Version

4.4

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Mac13.6

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Eism**
Resolves: #22193

### CODE REVIEW

File: SetupConfigure.cmake [hunk 1]
```diff
@@ -264,6 +265,7 @@ endfunction()
 
 # modules
 def_opt(MUE_BUILD_APPSHELL_MODULE ${MUE_BUILD_APPSHELL_MODULE})
+def_opt(MUE_BUILD_AUTOBOT_MODULE ${MUE_BUILD_AUTOBOT_MODULE})
```
Comment (igorkorsukov) [inline SetupConfigure.cmake:268]: this is not need

File: build/cmake/muse_framework/DeclareOptions.cmake [hunk 2]
```diff
@@ -25,6 +25,7 @@ declare_muse_module_opt(MUSESAMPLER ON)
 declare_muse_module_opt(NETWORK ON)
 declare_muse_module_opt(SHORTCUTS ON)
 declare_muse_module_opt(UI ON)
+declare_muse_module_opt(AUTOBOT ON)
```
Comment (igorkorsukov) [inline build/cmake/muse_framework/DeclareOptions.cmake:28]: please alphabet

File: build/cmake/muse_framework/DeclareOptions.cmake [hunk 3]
```diff
@@ -12,6 +12,7 @@ declare_muse_module_opt(AUDIO ON)
 option(MUSE_MODULE_AUDIO_JACK "Enable jack support" OFF)
 option(MUSE_MODULE_AUDIO_EXPORT "Enable audio export" ON)
 
+declare_muse_module_opt(AUTOBOT ON)
```
Comment (igorkorsukov) [inline build/cmake/muse_framework/DeclareOptions.cmake:15]: need also add here https://github.com/musescore/MuseScore/blob/master/build/cmake/muse_framework/Modules.cmake

**Review by igorkorsukov — APPROVED** [review]

**Comment by DmitryArefiev** [comment]
Tested #22193 on Win10, **Mac13.6**, LinuxUbuntu22.04 - **FIXED**

**Pull request merged**